csharp-intermediate-topics/ConstVsReadonlyInCSharp/ConstVsReadonlyInCSharp/CircleCalculator.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ConstVsReadonlyInCSharp
+{
+    public class CircleCalculator
+    {
+        public const double E = 2.71;
```

Comment:
This is never used nor called out in the article. Maybe it would be better to just remove it.

Reply:
I use it in my article as an example of how to declare a public const

Reply:
It is in the code listing and the previous paragraph mentions the idea, but as it is not being explicitly called out after the code listing and not being used anywhere, it seems like noise. A better location for it, would be in the MathCalculation class and then could be used. You could just have a method in the program app that performs a calculation with E to show how it is being used. It isn't related to circle calculations (or at least the ones being demonstrated). Even within the article, which I read, I didn't really see E being referred to and when looking at the code in the article I was trying to figure out what it was doing and how it related to a circle. Just my two cents, your editor will make the final call.